lib/DxilPIXPasses/DxilDebugInstrumentation.cpp

```diff
@@ -793,7 +929,8 @@ void DxilDebugInstrumentation::addDebugEntryValue(BuilderContext &BC,
                      WriteMask_X});
 
     m_RemainingReservedSpaceInBytes -= 4;
-    assert(m_RemainingReservedSpaceInBytes < 1024); // check for underflow
+    assert(m_RemainingReservedSpaceInBytes <
+           1024 * 1024); // check for underflow
```

Comment:
Does this mean we could reserve smaller space?

Reply:
The m_RemainingReservedSpaceInBytes is expected to be exact for the calculated debug payload for the given block. 
So, yes, this assert would be wrong if we tried to instrument a block with > 1M payload. Good catch! I'll reformulate that to be more obvious.